### PR TITLE
Urls simplification

### PR DESCRIPTION
**Pull request by vinyll**
Massively simplified URLs and templates.

### CODE REVIEW

**Comment by davidbgk** [comment]
It's a bit hard to review PRs built on top of others, can you merge the previous ones before?

File: register/views.py [hunk 1]
```diff
@@ -14,7 +16,12 @@
 
 
 class ProfileView(TemplateView):
-    template_name = 'register/profile.html'
+    template_name = 'registration/profile.html'
+
+    def get(self, request, *args, **kwargs):
```
Comment (yohanboniface) [inline register/views.py]: `login_required` ?

File: register/views.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
-from django.shortcuts import reverse
 from django.contrib import messages
 from django.contrib.auth import get_user_model, login, logout
 from django.contrib.sites.shortcuts import get_current_site
+from django.views.generic import TemplateView, FormView, RedirectView
```
Comment (yohanboniface) [inline register/views.py]: Duplicated imports?

File: register/templates/registration/email.html [hunk 3]
```diff
@@ -1,9 +1,10 @@
 {% extends "registration/registration_base.html" %}
 {% load i18n %}
 
-{% block title %}{% trans "Enter your email" %}{% endblock %}
+{% block title %}{% trans "Login or create an account" %}{% endblock %}
 
 {% block content %}
+<h1>Login or create an account</h1>
```
Comment (yohanboniface) [inline register/templates/registration/email.html:7]: I'd make the string as translatable, even if the i18n workflow is not yet finalized.

**Review by yohanboniface — CHANGES_REQUESTED** [review]
Also needs rebase after last merge :)

**Review by yohanboniface — APPROVED** [review]

**Pull request merged**